src/vat.sol

```diff
@@ -50,6 +50,9 @@ contract Vat {
     mapping (address => uint256)                   public dai;  // [rad]
     mapping (address => uint256)                   public sin;  // [rad]
 
+    bytes32 public slab = 0x0;  // Maker Art IPFS Hash
+    bytes32 public rest = 0x0;  // First 32+8 bytes
```

Comment:
I see you removed the ability to set these from the constructor, are we just waiting for another commit where you will either hardcode values or add a setter?

Reply:
Will make another commit when we have the values